src/cleanlab_tlm/utils/vpc/chat_completions.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
TODO: Confirm this VPC version matches our SaaS API as closely as we are easily able to.  The analogous SaaS API is defined here:  

https://github.com/cleanlab/cleanlab-tlm/pull/75